scripts/ncmtest.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+
```

Comment:
Could we create a new folder /scripts/test and move this script there?

Reply:
Makes sense. I will do that.

Reply:
Are you waiting me to fix these in this branch/PR or can I do it after you merge into master?